checks/cmciii_lcp_water

```diff
@@ -57,7 +57,7 @@ def parse_cmciii_lcp_water(info):
 
 def inventory_cmciii_lcp_water(parsed):
     if parsed:
-        yield (None, {})
+        yield ("LCP", {})
```

Comment:
The check actually expects two different items. "IN" and "OUT". This can be found reading the checkman entry or going back in the git history by 7 years. The correct items are than.

```python
yield ("IN", {})
yield ("OUT", {})
```